content/project-news/welcome-gsoc-2021-students.md

```diff
@@ -0,0 +1,27 @@
+---
+title:       "Welcome GSoC 2021 Students!"
+author:      "Colin Finck"
+date:        2021-05-18
+---
+
+Another year, another round of Google Summer of Code!
+The ReactOS Project is participating in Google Summer of Code 2021 and welcomes the selected students.
+
+**He Yang** is a successful student from GSoC 2020, who is going to continue his dive into user-mode ReactOS this year by working on _iernonce.dll_.
+That library is responsible for _RunOnceEx_, a prominent Windows feature to run multiple commands once after a system reboot.
+Implementing that feature will benefit software depending on it.
+Examples include installers that need to perform certain cleanup and completion tasks after the first reboot.  
+He Yang will be mentored by George Bișoc and Hermès Bélusca.
+
+**The_DarkFire_** has already impressed us with his [ReactOS live coding sessions](https://www.twitch.tv/the_darkfire_) prior to GSoC and proposed to complete his work on Symmetric Multiprocessing (SMP).
```

Comment:
`**_The_Darkfire_**` renders oddly. DarkFire is in italic, and all underscores are eaten.

Reply:
I would like to officially state I am 100% okay with just my real name (Justin Miller) being used.
For one thing any PRs I do in this project have to use my real name anyway, the fact I used my nick in my GSoC was an accident too.

Reply:
Thanks for the clarification! Will change accordingly